lib/krane/cli/render_command.rb

```diff
@@ -18,7 +19,14 @@ def self.from_options(options)
         bindings_parser = ::Krane::BindingsParser.new
         options[:bindings]&.each { |b| bindings_parser.add(b) }
 
-        ::Krane::OptionsHelper.with_processed_template_paths(options[:filenames]) do |paths|
+        # never mutate options directly
+        filenames = options[:filenames].dup
+        filenames << "-" if options[:stdin]
+        if filenames.empty?
+          raise Thor::RequiredArgumentMissingError, 'Must provied a value for --filenames or --stdin'
```

Comment:
nit: Bring this in line with the deploy_command

Reply:
Oh, definitely should be changed. It will come in the next commit, together with lint fix :)